### PR TITLE
Added Jest/Enzyme unit tests for all client/React components

### PR DESCRIPTION
**Pull request by lancl**
@vikramraja1995, this is my PR for today. Thanks!

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -68,12 +75,12 @@ class App extends React.Component{
...
 
         <div className="columnDivider">
-          <h3>Photo Carousel</h3>
+          {/* <h3>Photo Carousel</h3> */}
           <PhotoCarousel productName={this.state.productName} />
         </div>
 
         <div>
-          <h3>SideBar</h3>
+          {/* <h3>Side Bar</h3> */}
```
Comment (vikramraja1995) [inline client/src/components/App.jsx:83]: Were these comments pushed by mistake, or where they intentional?

File: unit_test/sum.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+function sum(a, b) {
```
Comment (vikramraja1995) [inline unit_test/sum.js]: Is this file intentional?

File: unit_test/sum.test.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const sum = require('./sum');
```
Comment (vikramraja1995) [inline unit_test/sum.test.js]: Is this file intentional?

**Comment by vikramraja1995** [comment]
Your test files look great, nice job @lancl! The only thing that I thought of pointing out is that, if you set a variable within the describe function to equal the shallow rendered react component, then you can reuse that in all your tests instead of rendering it each time.

File: client/src/components/App.jsx (hunk 1 above)
Comment (lancl) [reply client/src/components/App.jsx:83]: These are my comments. I should probably make them look more like comments, by replacing '<h3>' with 'Note'.

File: unit_test/sum.js (hunk 2 above)
Comment (lancl) [reply unit_test/sum.js]: Yes, they are my files, to test how Jest works. Any suggestion on where to place them? If not, I will just move them to a separate folder then.

File: unit_test/sum.test.js (hunk 3 above)
Comment (lancl) [reply unit_test/sum.test.js]: Same as above comment.

**Comment by lancl** [comment]
Good suggestion! On creating a variable that is equal to the shallow rendered react component. I will do that today.

File: unit_test/sum.js (hunk 2 above)
Comment (vikramraja1995) [reply unit_test/sum.js]: Well usually you aren't supposed to commit non-related code to the project. I'd suggest having it in your local repo and adding the file paths to .gitignore if necessary. Otherwise it can be confusing for anyone looking at your code later on.

Comment (lancl) [reply unit_test/sum.js]: Sounds good! Will do.

**Comment by vikramraja1995** [comment]
Looks good!
